Admin/Client/src/api/wtAdminApi.tsx

```diff
@@ -1,6 +1,7 @@
-import axios from 'axios';
+import axios, { AxiosError } from 'axios';
 
 const requestPath = '/request';
+const authenticationPath = '/authentication';
```

Comment:
isnt it declared in the config file?

Reply:
as we spoke - it does but you can't import it because it's CRA